Makefile

```diff
@@ -173,6 +173,9 @@ build-test: all build/$(TEST_JAR)
 
 test: all build/$(TEST_JAR)
 	echo "Running tests against $(LIB_PROFILER)"
+	cp -r build build_copy # make a copy to allow us to test how different copies of async profiler interact with one another
+	rm -rf build/build_copy
+	mv build_copy build
```

Comment:
This juggling is needed for one specific test. Makefile should not really care - it's a job of a test.